### PR TITLE
3338 ai index page

### LINKED ISSUE #3338: Beta: Build the AI overview page
Copy doc: https://docs.google.com/document/d/1mJq8PxH8FVRVF9i1Xzh0f6M73eyjqZE4NaKYHRcRAKM/edit

### LINKED ISSUE #3466: Change illustration on ai/index page
Illustration has moved out of the hero section.
New section below hero with new illustration

Copy doc: https://docs.google.com/document/d/1mJq8PxH8FVRVF9i1Xzh0f6M73eyjqZE4NaKYHRcRAKM/edit#heading=h.ut5vkcuyk8sv

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add `/ai` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/ai>
- See that it matches the [copy doc](https://docs.google.com/document/d/1mJq8PxH8FVRVF9i1Xzh0f6M73eyjqZE4NaKYHRcRAKM/edit#)

## Issue / Card

Fixes #3338 
Fixes #3466

### CODE REVIEW

File: templates/ai/index.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
+{% extends "ai/base_ai.html" %}
+{% block title %}AI and Machine Learning on Ubuntu{% endblock %}
+{% block meta_description %}Ubuntu is the operating system used for most machine learning production and engineering. The widest range of AI/ML frameworks and the easiest hardware GPGPU acceleration setup make Ubuntu the first choice for companies exploring AI algorithms and analytics. Canonical provides a full range of AI/ML solutions and services, from training to Kubernetes-based ML workflow operations and integration.{% endblock %}
+
+
+{% block content %}
+<section class="p-strip--light is-deep is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>End-to-end AI / ML workflow on Ubuntu</h1>
```
Comment (pmahnke) [inline templates/ai/index.html]: the h1 is now 'AI / ML from workstation to production'

File: templates/ai/index.html [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+      <p>Ubuntu is the ideal platform to power your Artificial Intelligence ambitions &mdash; from developer workstations, to racks, to clouds and to the edge with smart connected IoT. Canonical&rsquo;s Kubeflow supports the most popular tools for machine learning &mdash; starting with JupyterHub and Tensorflow &mdash; in a standardised workflow running on Kubernetes.</p>
```
Comment (pmahnke) [inline templates/ai/index.html]: remove 'ideal'

**Comment by pmahnke** [comment]
we think move the google quote down and then update the text above like so
image

**Comment by pmahnke** [comment]
leave the nvidia logo

**Comment by steverydz** [comment]
@anasereijo Done

**Comment by frankiegranato** [comment]
- [ ] para break added to hero 
- [ ] lower case 'a' on headline: 'Kubeflow — AI on Kubernetes — Anywhere'
- [ ] replace appearance of 'Pioneer' with Discoverer Plus. 
- [ ] Footer cta is amended.

**Comment by anasereijo** [comment]
@steverydz looks good! 👍  just one small change on the hero section because the title changed 
screen-shot-2018-06-27-at-10 30 14

**Comment by steverydz** [comment]
@anasereijo Done

**Comment by frankiegranato** [comment]
- [ ] New quote replaces existing one (sorry!). See highlight in doc. 
- [ ] Please update page description (Mark edited it)

**Comment by steverydz** [comment]
@frankiegranato Done

File: navigation.yaml [hunk 3]
```diff
@@ -33,6 +33,13 @@ openstack:
           path: /openstack/thank-you
           hidden: True
 
+ai:
+  title: AI/ML
+  path: /ai
+  children:
+    - title: AI/ML
+      path: /ai
```
Comment (anthonydillon) [inline navigation.yaml]: We don't need this sub navigation.

File: templates/ai/index.html [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+        <a class="p-button--neutral" href="/ai/contact-us?product=ai-index">Call the experts</a>
+      </p>
+    </div>
+    <div class="col-5 u-hide--small u-vertically-center">
+      <img src="{{ ASSET_SERVER_URL }}7d33e0a1-AI+illustration+hero.svg" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
```
Comment (anthonydillon) [inline templates/ai/index.html]: Missing the "Develop your AI Models on Ubuntu" section.

File: templates/ai/index.html [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Consulting to get started, Managed Ops to keep you focused</h2>
+    <p>Turn on the taps with a workshop to understand the full stack of machine learning. Build a full pipeline from developer stations to your data center, to the public cloud. Canonical works with the leading companies to ensure you have the widest range of choices. First, start with one of our standard bare-metal Kubernetes service packages (Discoverer or Discoverer Plus) and then select the AI Add-on to unlock the benefits of AI on Kubernetes.</p>
+    <p><a href="/kubernetes">Learn more about our Kubernetes packages&nbsp;&rsaquo;</a></p>
+    <h3 style="max-width: 100% !important">AI add-on for Kubernetes Discoverer and Discoverer Plus</h3>
```
Comment (anthonydillon) [inline templates/ai/index.html]: Does this need `!important`

File: templates/ai/index.html [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+        <li class="p-list__item is-ticked">Preliminary AI Discovery</li>
+        <li class="p-list__item is-ticked">Development Assessment</li>
+        <li class="p-list__item is-ticked">Deploy and Operate Analysis</li>
+        <li class="p-list__item is-ticked">Finalize Initial AI Strategy</li>
+      </ul>
+      <p><a class="p-button--positive" href="/ai/contact-us?product=ai-index-assessment">Contact us</a></p>
+      <p><span class="p-heading--four">$40,000</span> one off fee</p>
+    </div>
+    <div class="col-4 p-card">
+      <h3 class="p-card__title">Full remote management</h3>
```
Comment (anthonydillon) [inline templates/ai/index.html]: Remove this card

**Comment by steverydz** [comment]
@frankiegranato @anasereijo I've incorporated the changes from https://github.com/canonical-websites/www.ubuntu.com/issues/3466 into this PR as it isn't merged yet

**Comment by anasereijo** [comment]
@steverydz looking nice! we just need now to make the hero section white and the new one light grey

**Comment by steverydz** [comment]
@anthonydillon @anasereijo Changes done

**Comment by frankiegranato** [comment]
LGTM (will you resolve the comments in the doc when you're done please?)

**Pull request merged**